### DISCUSSION TITLE
How can I load the values of form components into `event.url.searchParams`?

### DISCUSSION THREAD

**Discussion by calionauta** [yesvelte/yesvelte#526] [Q&A]
Apparently, the values of the components (e.g. FormAutcomplete, FormSlider) do not appear in `event.url.searchParams` when using the 'GET' method of the form. Currently, I have to create hidden inputs with the same values in order to include them in the URL and send them to the server.

I might be missing something.

**Comment by TheHadiAhmadi** [comment]
we didn't implemented this feature for FormAutocomplete yet, but for FormSlider (and Slider) you should pass `name` to `SliderKnob` component, because we use bind:value for `SliderKnob` and value of this component is important.

https://www.yesvelte.com/docs/form/form-slider#With_Form_(submit)

**Reply by TheHadiAhmadi** [reply]
in https://github.com/yesvelte/yesvelte/pull/529 I added support for name prop.

for multiple values, should we create multiple `<input type="value" name="something" value="..."/>`? (for example in checkbox group or Autocomplete multiple, ..)

**Reply by calionauta** [reply]
Thank you for #529 .

> for multiple values, should we create multiple <input type="value" name="something" value="..."/>? (for example in checkbox group or Autocomplete multiple, ..)

I think that the suggested alternative could be a good option. Maybe it would work almost like the `SELECT multiple` HTML element, where the same "field" is sent multiple times with the same name but with different values.

**Reply by TheHadiAhmadi** [reply]
Thanks, using Select multiple could be a better option 👍

**Comment by TheHadiAhmadi** [comment]
@renatocaliari Here you can see an example of form submission using `<form>` and `name` prop.

https://www.yesvelte.com/demo/form

Let me know if there is any problem.
Thanks

**Reply by calionauta** [reply]
Great. Thanks @TheHadiAhmadi
